ext/stringio/stringio.c

```diff
@@ -49,6 +49,13 @@ static long strio_write(VALUE self, VALUE str);
 #define IS_STRIO(obj) (rb_typeddata_is_kind_of((obj), &strio_data_type))
 #define error_inval(msg) (rb_syserr_fail(EINVAL, msg))
 #define get_enc(ptr) ((ptr)->enc ? (ptr)->enc : !NIL_P((ptr)->string) ? rb_enc_get((ptr)->string) : NULL)
+#ifndef HAVE_RB_STR_CHILLED_P
+static int
+rb_str_chilled_p(VALUE str)
+{
+    return 0;
```

Comment:
It seems that this is different from the original signature: https://github.com/ruby/ruby/pull/10355/files#diff-b1bbd2981f285b9ed71e1645ef2194ea2528699c342842756c0dafebe225013dR615

```suggestion
static bool
rb_str_chilled_p(VALUE str)
{
    return false;
```

Reply:
Changing to `bool` makes it fail under Ruby 2.7. Should we drop 2.7 from CI since it's EOL?
Or is there a better way to fix this?

Reply:
You can `#include <stdbool.h>`.

It's now done when you include ruby.h, but prior to 2.7 it wasn't.